design/baremetal-operator/bmc-events.md

```diff
@@ -79,15 +81,15 @@ status:
 
 - A BMCEventSubscription resource represents a subscription to the events generated
   by a specific BMC.
-- Ironic will manage configuring the subscription using a vendor passthru API.
```

Comment:
On the fence here: Ironic actually calls it "passthru".

Reply:
I can change that back if its an actual term and not just common shorthand.